util/fetch_shas.sh

```diff
@@ -54,54 +54,47 @@ enumerate_keys() {
   done
 }
 
-emit_bzl_file_contents() {
-  if which parallel > /dev/null; then
```

Comment:
This code path generated different content than the other path. It was removed because it's slower and there should be only one way to update known_shas.bzl

Reply:
> @wt Hey, do you know why the `known_shas.sh` looks so pruned after running `bazel run //util:fetch_shas`?

Found the issue and resolved it in d1620614353bbd6d65101f8cbe026b6a78ea9306. @illicitonion can you take a look at that commit and let me know what you think? I can move that into it's own PR if you'd prefer.

Reply:
This was resolved by https://github.com/bazelbuild/rules_rust/pull/1355